components/schemas/scoped-variables/ScopedVariableAccess.yml

```diff
@@ -0,0 +1,25 @@
+title: ScopedVariableAccess
+type: object
+description: The type or way the scoped variable is accessed.
+required:
+  - env_variable
+properties:
+  env_variable:
+    type: boolean
+    description: A boolean, where true represents this scoped variable is accessed like a normal environment variable.
```

Comment:
"When set to true, this scoped variable is set as an environment variable inside the container."